core/src/main/scala/kanaloa/reactive/dispatcher/queue/QueueProcessor.scala

```diff
@@ -65,7 +65,6 @@ class QueueProcessor(
 
     case Terminated(worker) if workerPool.contains(worker) ⇒
       removeWorker(worker)
-      healthCheck()
```

Comment:
This is what stops the spin, right?  However, this still gets executed on a scheduled interval, so if all Workers do fail, at some point, the minimum number of Workers will attempted to be created, right?